### DISCUSSION TITLE
1/2, 1/4, and 1/8 Speed Clocks

### DISCUSSION THREAD

**Discussion by OmniacDev** [SebLague/Digital-Logic-Sim#94] [Show and tell]
Hey everyone back again with some more clocks. In #92 I showed how you could make a simple clock, but they ran a bit too fast for my liking, so I tinkered around and figured out how to make a slower one.

Building these slower clocks follow the same rules as when we built the simple clock in #92. They need to be built in an exact order or else they won't work.

### 1/2 Speed Clock

This clock is about 1/2 the speed of the original clock, though it's kind of hard to tell because of how fast the clock goes. To build this please follow the steps below in the exact order they are shown.

1. First create an input pin called "IN".
2. Next, create an output pin called "OUT".
3. Drag in 2 "AND" circuits.
4. Drag in 2 "CLOCK" or "CLK" circuits. (See #92 on how to build these).
5. Drag in 2 "NOT" circuits.

Now you can wire them up as shown in the video below:

https://user-images.githubusercontent.com/64994204/157108546-a6453abc-d687-4107-897b-2bd270e4a9d1.mp4

### 1/4 Speed Clock

This clock is about 1/4 of the speed of the original clock. Again, please follow the exact steps for this to work.

1. First create an input pin called "IN".
2. Next, create an output pin called "OUT".
3. Drag in 2 "AND" circuits.
4. Drag in 2 "1/2 CLOCK" or "1/2 CLK" circuits that we built earlier.
5. Drag in 2 "NOT" circuits.

Now you can wire the up as shown in the video:

https://user-images.githubusercontent.com/64994204/157108889-033589ea-bbd7-4b27-9668-81c5d2238283.mp4

### 1/8 Speed Clock

This clock is about 1/8 of the speed of the original clock. Again, please follow the exact steps for this to work.

1. First create an input pin called "IN".
2. Next, create an output pin called "OUT".
3. Drag in 2 "AND" circuits.
4. Drag in 2 "1/4 CLOCK" or "1/4 CLK" circuits that we built earlier.
5. Drag in 2 "NOT" circuits.

Now you can wire the up as shown in the video:

https://user-images.githubusercontent.com/64994204/157108971-3fc1fa7f-fc37-42fa-b502-95e1f4b012df.mp4

Thanks for reading, I hope you found what you needed.

**Comment by Queen-Elizabeth-II** [comment]
To make the clocks even slower just do the exact same but with the clock you just made, that way you can make 1/32 or 1/16 clocks
